docs/animations.md

```diff
@@ -36,7 +36,7 @@ const FadeInView = (props) => {
     <Animated.View                 // Special animatable View
       style={{
         ...props.style,
-        opacity: fadeAdnim,         // Bind opacity to animated value
+        opacity: fadeAdmin,         // Bind opacity to animated value
```

Comment:
You made me realized that it should be `fadeAnim`, not `fadeAdmin`. 

Are you up to replace it elsewhere as well?